app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        name_str = f"Name: {self.name}, "
+        health_str = f"Health: {self.health}, "
+        hidden_str = f"Hidden: {self.hidden}"
+        return "{" + name_str + health_str + hidden_str + "}"
```

Comment:
no need to create this variables here, use brackets to break the line

Reply:
but otherwise, it will come this way, which was in bad practice examples
![image](https://user-images.githubusercontent.com/123904633/218478958-bd011860-30fe-4f9b-b437-1d50f0060a66.png)